tutorials/evoked/plot_evoked_measures.py

```diff
@@ -0,0 +1,153 @@
+"""
+.. _tut_erp:
+
+Event Related Potentials (ERPs) Measures
+==================================================
+
+.. contents:: Here we cover the specifics of extracting data from ERPs, namely:
+   :local:
+   :depth: 1
+
+"""
+
+import pandas as pd
+import matplotlib.pyplot as plt
+
+import mne
+from mne.datasets import sample
+
+###############################################################################
+# Create evoked from raw data
+
+# Read in raw data
+data_path = sample.data_path()
+raw_fname = data_path + '/MEG/sample/sample_audvis_filt-0-40_raw.fif'
+event_fname = data_path + '/MEG/sample/sample_audvis_filt-0-40_raw-eve.fif'
+raw = mne.io.read_raw_fif(raw_fname, preload=True)
+
+# Select only the the EEG channels
+raw.pick_types(meg=False, eeg=True, eog=True).load_data()
+
+# Create epochs from events, and reject epochs
+events = events = mne.read_events(event_fname)
+event_id = {'auditory/left': 1,
+            'auditory/right': 2,
+            'visual/left': 3,
+            'visual/right': 4}
+tmin, tmax = -0.2, 0.5
+reject = dict(eeg=180e-6, eog=150e-6)
+epochs_params = dict(events=events, event_id=event_id, tmin=tmin, tmax=tmax,
+                     reject=reject)
+epochs = mne.Epochs(raw, **epochs_params)
+
+# Plot the sensors to show locations
+epochs.plot_sensors(show_names=True)
+
+# Make all evokeds
+evokeds = dict((cond, epochs[cond].average()) for cond in event_id)
+print(evokeds['visual/left'])
+
+# Plot the evokeds for visual trial
+evokeds['visual/left'].plot_joint(title='Visual Left (EEG)',
+                                  times=[.09, .18, .27])
+evokeds['visual/right'].plot_joint(title='Visual Right (EEG)',
+                                   times=[.09, .18, .27])
+
+###############################################################################
+# Obtaining Peak Latency and Amplitude
+# ---------------------
+#
+# One of the most common ERP measures is optaining the peak amplitude and time.
+# This usually is done by searching in a user-specifed time frame. We
+# will obtain the peak latency and amplitude for the first positive peak for
+# the visual trials in each hemisphere. The :func:`mne.io.Evoked.get_peak`
+# method to achieve this. Here, we will focus on channels 'EEG 057' and
+# 'EEG 059' (left and right hemispheres, respectively).
+#
+# To mitigate high-frequency noise influencing peak measures, we can low-pass
+# filter the data with a cutoff of 20Hz or 30Hz. Here, we will do 20Hz. Note
+# that the Raw object of this sample dataset is lowpass filtered at 40Hz.
+
+# First we will xxtract the data from right trials in contralateral hemisphere
+
+# Define channel to use
+chan = 'EEG 054'
+
+# Copy and filter evoked
+right_vis_evoked = evokeds['visual/right'].copy()
```

Comment:
Maybe just:
```suggestion
right_vis_evoked = evokeds['visual/right'].copy().pick(chan).filter(None, 20)
```
(if it fits in one line well)

Reply:
(BTW the `.copy()` is not necessary here)

Reply:
thanks for pointing that out. In an effort to simplify the tutorial, I'm drop the bit about the `filter()` stuff, mainly because the data are so clean (even with the 40Hz low-pass, there is no noticeable noise).